config/__init__.py

```diff
@@ -2,13 +2,31 @@
 
 FLASK_ENV = environ.get("FLASK_ENV")
 
-if FLASK_ENV == "development":
-    from config.environments.development import DevelopmentConfig as Config
-elif FLASK_ENV == "dev":
-    from config.environments.dev import DevConfig as Config
-elif FLASK_ENV == "test":
-    from config.environments.test import TestConfig as Config
-elif FLASK_ENV == "production":
-    from config.environments.production import ProductionConfig as Config
-else:
-    from config.environments.default import Config  # noqa
+match FLASK_ENV:  # noqa
```

Comment:
⛏️ we could put a `#flask8 : noqa` at the top of this file, it will still be formatted by black but this redefinition error will be ignored.

Reply:
noice - slipped by that one 🐍